docker/docker_image_dest.go

```diff
@@ -236,7 +236,7 @@ func (d *dockerImageDestination) PutManifest(m []byte) error {
 		return err
 	}
 	defer res.Body.Close()
-	if res.StatusCode != http.StatusCreated {
+	if successStatus(res.StatusCode) {
```

Comment:
Wait, shouldn't this be `!successStatus`?

(I do hope that that will show up in the CI.)